packages/core/src/components/graphs/skeleton-lines.ts

```diff
@@ -29,32 +29,32 @@ export class SkeletonLines extends Skeleton {
 		svg.attr("width", width).attr("height", height);
 
 		const isDataLoading = Tools.getProperty(
-			this.model.getOptions(),
+			this.getOptions(),
```

Comment:
why not grab options from model?

Reply:
gets the this.modelgetOptions > merges it with any component specific options then returns it